src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/network-layer-protection/network-list.mdx

```diff
@@ -6,21 +6,21 @@ namespace: documentation_products_edge_firewall_network_lists
 permalink: /documentation/products/secure/edge-firewall/network-layer-protection/network-lists/
 ---
 
-**Network Lists** is a feature from [Network Layer Protection](/en/documentation/products/edge-firewall/network-layer-protection/), an **Edge Firewall** module. With Network Lists, you can create and manage *allowlists*, *blocklists*, and even *greylists*, based on the user's network, *IP address*, geolocation (*countries*), or *Autonomous System Number (ASN)*. With this feature, it's possible to prevent different types of attacks to your network, as well as prevent users with malicious behavior from accessing your applications.
+**Network Lists** is a feature from [Network Layer Protection](/en/ddocumentation/products/secure/edge-firewall/network-layer-protection/), an **Edge Firewall** module. With Network Lists, you can create and manage *allowlists*, *blocklists*, and even *greylists*, based on the user's network, *IP address*, geolocation (*countries*), or *Autonomous System Number (ASN)*. With this feature, it's possible to prevent different types of attacks to your network, as well as prevent users with malicious behavior from accessing your applications.
```

Comment:
```suggestion
**Network Lists** is a feature from [Network Layer Protection](/en/documentation/products/secure/edge-firewall/network-layer-protection/), an **Edge Firewall** module. With Network Lists, you can create and manage *allowlists*, *blocklists*, and even *greylists*, based on the user's network, *IP address*, geolocation (*countries*), or *Autonomous System Number (ASN)*. With this feature, it's possible to prevent different types of attacks to your network, as well as prevent users with malicious behavior from accessing your applications.
```